app/templates/show.hbs

```diff
@@ -39,7 +39,11 @@
         </div>
       {{else}}
         <div class="additional-info">
-          {{vod-chapters chapters=show.vods.firstObject.chapters seekto=seekto setSeekTo=(action 'setSeekTo')}}
+          {{#if show.vods.firstObject.chaptersPublished}}
+            {{vod-chapters chapters=show.vods.firstObject.chapters seekto=seekto setSeekTo=(action 'setSeekTo')}}
+          {{else}}
+            <h5>There are no chapters currently published for this show.</h5>
```

Comment:
We shouldn't display this on the public site. A show with unpublished chapters should just act as no chapters at all.